src/Avalonia.Controls.TreeDataGrid/ITreeDataGridSource.cs

```diff
@@ -31,6 +31,18 @@ public interface ITreeDataGridSource
         /// </summary>
         ITreeDataGridSelection? Selection { get; }
 
+        /// <summary>
+        /// Returns all the items in the source.
```

Comment:
Nit: use the same wording as in `ITreeDataGridSource<TModel>`:

```suggestion
        /// Gets the items in the data source.
```